gcloud/monitoring/client.py

```diff
@@ -195,6 +203,130 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
+
+             >>> type_ = 'compute.googleapis.com/instance/cpu/utilization'
+             >>> metric = client.metric(type_, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        For custom metrics, the simplest way to get the type is from the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` class used to
+        create the custom metric::
```

Comment:
I suggest simply specifying the custom metric type name as a literal, as you have done above for the built-in type. Talking about getting it from the MetricDescriptor instance used to write the custom metric [descriptor] is confusing, because the metric descriptor might have been created a long time ago, in other software.